xdsl/dialects/llvm.py

```diff
@@ -474,6 +475,64 @@ def print(self, printer: Printer) -> None:
         printer.print(self.lhs.type)
 
 
+class ArithmeticBinOpExact(IRDLOperation, ABC):
+    """Class for arithmetic binary operations that use an exact flag."""
+
+    T: ClassVar = VarConstraint("T", BaseAttr(IntegerType))
+
+    lhs = operand_def(T)
+    rhs = operand_def(T)
+    res = result_def(T)
+    is_exact = opt_prop_def(BoolAttr, prop_name="isExact")
```

Comment:
Oh, one more thing, is that I don't think it makes sense for this property to be optional, isn't it the same as default 0?
```suggestion
    is_exact = prop_def(BoolAttr, prop_name="isExact")
```

Reply:
also, `default_value` might be useful here